eslint.config.js

```diff
@@ -0,0 +1,40 @@
+"use strict";
+
+const eslintConfigESLint = require("eslint-config-eslint");
+const globals = require("globals");
+
+module.exports = [
+    ...eslintConfigESLint,
+    {
+        languageOptions: {
+            sourceType: "commonjs",
+            ecmaVersion: 2018,
+            globals: {
+                ...globals.node,
+                ...globals.es2015
```

Comment:
This isn't needed, as globals are automatically added due to `ecmaVersion: 2018`